specs/architecture.html

```diff
@@ -358,7 +358,7 @@
       	  <tr>
       	    <td>Payment Mediator</td>
       	    <td>Digital Wallet, Payment Instrument</td>
-      	    <td>A piece of software that coordinates messages between Web apps and Payment apps</td>
+      	    <td>A piece of software that coordinates messages between sellers and Payment apps</td>
       	  </tr>
```

Comment:
Don't agree with this change. The messages are exchanged between the website and the payment app via the mediator.